src/pages/kb/user-guide/visualizations/visualizations-how-to.md

```diff
@@ -9,40 +9,30 @@ slug: visualizations-how-to
 
 ## Create a New Visualization
 
-Once your query has finished running for the first time, you can add a
-visualization for your results by clicking the “New Visualization” button
-above the results (currently shown only when the source code is shown).
+Once your query has finished running for the first time, you can add a visualization by clicking the “New Visualization” button above the results table.
 
-![](/assets/images/docs/gifs/visualization/new_viz.gif)
+![](/assets/images/docs/gitbook/new-viz.png)
 
 ## Edit A Visualization
 
-To change an existing widget's parameters, enter the query of that widget
-(just click its title). Then, view the source code and you'll see an "Edit"
-option under each visualization, clicking it will open the current settings
-for that visualization (type, x, y, group...).
-
-<img src="/assets/images/docs/gitbook/edit_viz.png" width="60%">
-
-Hit "Save" to apply your changes or "Cancel" to leave no trace.
+You can modify the controls for an existing visualization from the query editor screen. Click the visualization on the tab bar and you'll see an `Edit Visualization` option beneath each visualization. Clicking it will open the current settings for that visualization (viz type, X axis, Y axis, groupings etc.). Hit "Save" to apply your changes or "Cancel" to leave no trace.
 
 ## Embedding Visualizations
 
-For any visualization you create in Redash, you can get an embed link:
+It's easy to embed Redash visualizations. Just click the elipsis button beneath any visualization to show further options and select `Embed Elsewhere`.
```

Comment:
To make reality inline with docs: https://github.com/getredash/redash/pull/3844 :-) It actually bugged me for sometime.

Reply:
Fixed my screenshot to show proper capitalization in  cadbacd4fc591108d52cdb70d90f7a88e839fbb7